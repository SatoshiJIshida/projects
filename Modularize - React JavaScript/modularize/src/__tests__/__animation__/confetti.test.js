import React from "react";
import { shallow } from "enzyme";
import ConfettiAnimation from "../../components/animation/confetti";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Confetti Animation", () => {
  it("should be defined", () => {
    expect(ConfettiAnimation).toBeDefined();
  });

  it("should render correctly", () => {
    const img = shallow(<ConfettiAnimation name="img-test" />);
    expect(img).toMatchSnapshot();
  });
});
