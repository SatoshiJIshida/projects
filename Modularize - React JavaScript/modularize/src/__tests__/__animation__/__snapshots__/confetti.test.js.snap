// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confetti Animation should render correctly 1`] = `ShallowWrapper {}`;
